pkg/plugins/datasource_plugin.go

```diff
@@ -62,7 +62,11 @@ func (p *DataSourcePlugin) startBackendPlugin(ctx context.Context, log log.Logge
 
 	err := p.spawnSubProcess()
 	if err == nil {
-		go p.restartKilledProcess(ctx)
+		go func() {
+			if err := p.restartKilledProcess(ctx); err != nil {
+				// TODO: Deal with error
```

Comment:
Seems like `restartKilledProcess` only returns when context is done, i.e. returns `ctx.Err()` so not sure what to do here. Maybe not return an error at all? Or log it.

Reply:
@marefr I agree with your analysis, made it log the error.